lib/robots/dor_repo/release/release_members.rb

```diff
@@ -35,24 +32,17 @@ def publish_collection(druid, item)
           #   if at least one of the targets is *not* what=self, we will do it
           tag_service = Dor::ReleaseTagService.for(item.object)
           release_tags = tag_service.newest_release_tag(tag_service.release_tags) # get the latest release tag for each target
-          if release_tags.collect { |_k, v| v['what'] == 'self' }.include?(false) # if there are any *non* what=self release tags in any targets, go ahead and add the workflow to the items
-            add_workflow_to_members(member_service, item.object_type)
-          else # all of the latest release tags are what=self or there are no release tags, so skip
-            LyberCore::Log.debug "...all release tags are what=self for #{item.object_type}; skipping member workflows"
-          end
+          # if there are any *non* what=self release tags in any targets, go ahead and add the workflow to the items
+          add_workflow_to_members(member_service) if release_tags.collect { |_k, v| v['what'] == 'self' }.include?(false)
```

Comment:
I suspect there's a more streamlined way to detect if any `'what'` values are not `'self'` but this is not related to removing the debug call so `/shrug`